rabbitmq.py

```diff
@@ -92,7 +92,8 @@ def _api_call(self, url):
         # RabbitMQ doesn't mind.  The alternative is globally modifying the
         # urllib with an opener each time before making a request.
         req.add_header("Authorization", "Basic %s" %
-                       base64.b64encode('%s:%s' % (self.username, self.password)))
+                       base64.b64encode('%s:%s' % (self.username,
```

Comment:
Can't we just crank the line length up to something more reasonable like 100 or 120? We don't have 80 character terminals anymore.